src/functions/events/v2.ts

```diff
@@ -1,4 +1,4 @@
-export const PUBSUB_PUBLISH_EVENT = "google.cloud.pubsub.topic.v1.messagePublished" as const;
```

Comment:
Do you know why `as const` was there in the first place? Was it used as a string literal type somewhere & isn't now?

Reply:
I'm not sure why we had it like this, maybe leftover from a different implementation?